apps/aechannel/src/aesc_fsm.erl

```diff
@@ -2262,8 +2321,8 @@ send_inband_msg(To, Info, #data{session     = Session} = D) ->
     aesc_session_noise:inband_msg(Session, M),
     log(snd, ?INBAND_MSG, M, D).
 
-check_inband_msg(_, #data{ client_connected = false }) ->
-    {error, client_disconnected};
+%% check_inband_msg(_, #data{ client_connected = false }) ->
+%%     {error, client_disconnected};
```

Comment:
```suggestion
     {error, client_disconnected};
```

Reply:
Since inband messages don't require co-signing, it's actually appropriate to do the test here.